content/en/blog/hacktoberfest-2021/index.md

```diff
@@ -0,0 +1,119 @@
+
+## Do you have any suggestions on what to contribute?
+
+<img class="offset-image offset-image-right" src="/card-icons/code.svg" height="150px" width="190px" style="height: 150px; margin-right: -100px; margin-top: -50px;" alt="">
+
+If you're new to Datenanfragen.de, you may be a little overwhelmed and don't know where to start. Don't worry: There are numerous areas in which you can participate—regardless of whether you have experience with programming and data protection or not. To make it easier for you to get started, we have put together a few suggestions here. You can find many more tasks in the issues of the respective repositories. Of course we also welcome your own ideas on how you could improve the project.
+
+* **Collecting suggested companies for more countries**  
+  As explained above, we want to collect lists of companies that users should request their data from. We need these lists to suggest these companies by our wizard to users in these countries. At the moment we support Germany, Austria, and Great Britian. However, we would like to support all EU countries in the future. Read more in [this GitHub issue](https://github.com/datenanfragen/data/issues/230).
+
+* **Add required elements to company records**  
+  Companies have to identify to respond to your GDPR requests. However, different companies require different methods of identification. That's why we collect the required identification elements for companies.
+
+  As we only have this data for some companies, we default to name, email, and address for many companies. But those fields aren't optimal for most companies, some require more, some less data. This is where we need your help: [This GitHub issue](https://github.com/datenanfragen/data/issues/720) explains in detail how you can add the missing data by sending your own GDPR requests to the corresponding companies. As a side effect, you'll also learn something about the data those companies process on you.
+
+* **Writing and translating blog post**  
+
+  We have several entries in our {< link slug="blog" text="blog" >}}. They cover a wide spectrum from explanation of GPDR laws, to tutorials and reports. We want to publish more articles and are happy about fitting submissions to the [website repository](https://github.com/datenanfragen/website). 
+
+  Another important task is translating existing posts into the other languages that we support. The [website repository](https://github.com/datenanfragen/website) is the right place to go for that, as well.
+
+* **Translating the request templates to more languages**  
+  Our request templates are already available in many languages but some are outdated, as we already updated the English and German templates. Help us by translating these updates to other langauges as well. Learn more in [this GitHub issue](https://github.com/datenanfragen/data/issues/229).
```

Comment:
```suggestion
  Our request templates are already available in many languages but some are outdated, as we have since updated the English and German templates. Help us by translating these updates to other languages as well. Learn more in [this GitHub issue](https://github.com/datenanfragen/data/issues/229).
```